frontend/src/lib/i18n/index.js

```diff
@@ -1,15 +1,21 @@
-import { browser } from '$app/environment';
+import {} from '$app/environment';
```

Comment:
What's the purpose of this line?

Reply:
> What's the purpose of this line?

I dont remmeber why and how i did it. Im almost sure it was occasionally